src/hexo/themes/documentation/layout/partials/home.hbs

```diff
@@ -9,16 +9,58 @@
             </div>
         </div>
         <div class="dark-blue">
-            <div class="home-container container">
-                <section class="home-container home-container--section">
-                    <h1>sonar</h1>
-                    <p class="subheading">A linting tool for the web.</p>
+            <div class="container home-container">
+                <section class="home-container--section">
+                    <h1 class="headline">sonar</h1>
+                    <p class="title">a linting tool for the web</p>
                     <div class="actions">
                         <a class="button button-cta" href="/docs/user-guide/">Getting Started</a>
                     </div>
                 </section>
-                <section class="home-container home-container--section">
-                    <h2 class="title uppercase-text">Install sonar (requires node 8)</h2>
+                <section class="home-container--section home-container__features">
+                    <h2 class="subheading subheading--home">Build better websites with sonar</h2>
+                    <div class="layout layout--quarters">
+                        <div class="module">
+                            <p class="subtitle--alt subtitle--home uppercase-text">Accessibility, security, interoperability and more!</p>
+                            <p class="feature_highlight">
+                                The web is complex. Knowing everything about web development and not making any mistakes can be challenging.
+                                sonar is meant to help developers regardless of their skill level, from the novice to the
+                                experienced one, to build websites and apps that work for everyone, are accessible, fast, secure, and
+                                take advantage of all the features the web has to offer.
+                            </p>
+                        </div>
+                        <div class="module">
+                            <p class="subtitle--alt subtitle--home uppercase-text">Guidance tailored to your needs</p>
+                            <p class="feature_highlight">
+                                An online shopping experience probably doesn't have the same requirements as an intranet website.
+                                sonar is useful no matter what you are working on, and will provide you the best guidance for
+                                your needs. See how sonar can adapt its feedback when you give it more information about the resources
+                                you [cannot control](sectionNeedsAHeader), the <a href="/docs/user-guide/#browserconfiguration">browsers your visitors use</a>,
```

Comment:
We need a hyperlink here @molant which is what I assume the (sectionsNeedsAHeader) is in reference to.

Reply:
@ststimac @molant https://sonarwhal.com/docs/user-guide/#ruleconfiguration ?

Reply:
I've [created a new one](https://github.com/sonarwhal/sonar/commit/c2e2ca0a8313942e338c59358941dd5fa0c22da5), waiting for it to deploy

Reply:
> waiting for it to deploy

[Done](https://sonarwhal.com/docs/user-guide/#ignoringdomains).

Reply:
I've fixed this before merging.